Assets/HoloToolkit/Input/Scripts/GamePad/XboxController/XboxControllerHandlerBase.cs

```diff
@@ -56,22 +49,21 @@ public virtual void OnXboxInputUpdate(XboxControllerEventData eventData)
                 GamePadName = eventData.GamePadName;
             }
 
-            if (XboxControllerMapping.GetButton_Down(SelectButton, eventData))
+            if (eventData.XboxA_Down)
             {
+                CurrentGestureState = GestureState.SelectButtonPressed;
+
                 InputManager.Instance.RaiseSourceDown(eventData.InputSource, eventData.SourceId, InteractionSourcePressInfo.Select);
+
+                HoldStartedRoutine = StartCoroutine(HandleHoldStarted(eventData));
             }
 
-            if (XboxControllerMapping.GetButton_Pressed(SelectButton, eventData))
+            if (eventData.XboxA_Pressed)
             {
                 HandleNavigation(eventData);
-
-                if (!HoldStarted && !RaiseOnce && !NavigationStarted)
-                {
-                    HandStartedRoutine = StartCoroutine(HandleHoldStarted(eventData));
-                }
             }
 
-            if (XboxControllerMapping.GetButton_Up(SelectButton, eventData))
+            if (eventData.XboxA_Up)
```

Comment:
this removes the customization of the selection buttons

Reply:
Ahh yes, this was left over from when I was investigating the double events. Reverting.